app/Hutch.Relay/Hutch.Relay.csproj

```diff
@@ -9,7 +9,6 @@
     <UserSecretsId>30efce71-708c-4ab7-9d7b-8deab52cc1bf</UserSecretsId>
```

Comment:
It would be helpful to do a version bump in this so we can do a release, or fine in a new PR too

Reply:
(will leave it up to you though)